src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        putValue(hash(key), key, value);
     }
 
     @Override
     public V getValue(K key) {
+        int position = hash(key);
+        Node<K, V> node = table[position];
+        while (node != null) {
+            if (Objects.equals(key, node.key)) {
+                return node.value;
+            }
+            node = node.next;
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private void putValue(int index, K key, V value) {
+        if (size > (table.length * LOAD_FACTOR)) {
```

Comment:
Useless brackets, cause arithmetic operation more valuable then boolean operation 
```suggestion
        if (size > table.length * LOAD_FACTOR) {
```